code/centauri/hyperspace/default.nix

```diff
@@ -0,0 +1,81 @@
+{ pkgs, crane, common-attrs, common-deps }: {
+
+  template = { relaychainHostA ? "127.0.0.1", relaychainPortA ? 9944
+    , parachainHostA ? "127.0.0.1", parachainPortA ? 9988, paraIdA ? 2001
+    , commitmentPrefixesA ? "0x6962632f", clientIdA ? "11-beefy-0"
+
+    , relaychainHostB ? "127.0.0.1", relaychainPortB ? 9944
+    , parachainHostB ? "127.0.0.1", parachainPortB ? 9188, paraIdB ? 2000
+    , commitmentPrefixesB ? "0x6962632f", clientIdB ? "11-beefy-0", }: {
+      bin = crane.buildPackage (common-attrs // {
+        name = "hyperspace";
+        pname = "hyperspace";
+        cargoArtifacts = common-deps;
+        cargoExtraArgs = "--package hyperspace";
+        installPhase = ''
+          mkdir --parents $out/bin
+          cp target/release/hyperspace $out/bin/hyperspace
+        '';
+        meta = { mainProgram = "hyperspace"; };
+      });
+      rawConfig = (builtins.fromTOML
+        (builtins.readFile ./code/centauri/hyperspace/config.toml));
```

Comment:
Does this setup overwrite what's in the `config.toml` file

Reply:
We have to overwrite so we can target release to different hosts/ports/names/etc. So it is only builder now. Please specify any combo you need for later work.